schema/planned-expenses.js

```diff
@@ -0,0 +1,25 @@
+import { gql } from "apollo-server";
```

Comment:
there is a chart schema and resolver already present. This is another data set for a chart, you can add to the existing schema and resolver of charts